syntax = "proto3";

message StoreRequest {
  string key = 1;
  string value = 2;
}